Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -39,16 +41,53 @@ AddNewCardViewModelOutputs {
 
   public init() {
     let cardholderName = self.cardholderNameChangedProperty.signal.skipNil()
-    let creditCardDetails = self.creditCardChangedProperty.signal.skipNil()
+    let creditCardDetails = self.creditCardChangedProperty
+      .signal
```

Comment:
Should we keep `.signal` at the end of previous line? Since we almost always deal with signals it's not needed to be as explicit as the transformations we do on it `skipNil`, `filter`, etc.

(just a suggestion)